tests/functional/aws-node-sdk/test/utils/init.js

```diff
@@ -0,0 +1,19 @@
+const metadata = require('../../../../../lib/metadata/wrapper');
+let metadataInit = false;
+
+function initMetadata(done) {
+	if (metadataInit === true) {
```

Comment:
simply `if (metadataInit) {` could be cleaner

Reply:
i like the fact of strict checking personnaly :)